lib/node_modules/@stdlib/array/log/docs/repl.txt

```diff
@@ -0,0 +1,21 @@
+{{alias}}( str[, ...args] )
+    Inserts array element values into a format string and prints the result.
+
+    Parameters
+    ----------
+    str: str
+        Format string.
+
+    ...args: Collection|*
```

Comment:
Need to update how variadic args are documented and use `any`. See https://github.com/stdlib-js/stdlib/blob/develop/docs/repl_text.md.